micmac_li3ds/api.py

```diff
@@ -7,6 +7,14 @@
 class Api(object):
 
     def __init__(self, api_url, api_key):
+        if not api_url:
+            err = 'Error: no api url provided'
+            raise ValueError(err)
+
+        if not api_key:
+            err = 'Error: no api key provided'
+            raise ValueError(err)
+
```

Comment:
Minor, but I don't think we need to bother checking that `api_url` and `api_key` are not `None` here.